<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sqlite</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>学生学号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="idEdit">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>学生姓名：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="nameEdit">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>学生成绩：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="scoreEdit">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="insertButton">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>插入</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateButton">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectButton">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
